101-media-services-create/azuredeploy.json

```diff
@@ -0,0 +1,57 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "name": {
```

Comment:
"name" is ambiguous and may cause problems with cmd line tools - use mediaServiceName or some qualifier